### ISSUE TITLE
Incorrect Source Size

### ISSUE THREAD

**Issue by williamjohnstone** [endevrr/AutoRclone-Rewrite#9]
AutoRclone is displaying the incorrect source size due to this line in `autorclone.py` 
```python
amount_to_transfer_bytes = int(helpers.calculate_path_size(src_label, rclone_generated_config_path)) - int(helpers.calculate_path_size(dst_label, rclone_generated_config_path))
```
This was originally written under the assumption all users are mirroring drives, this is obviously not the case and I don't know what I was thinking when I approved the change, I'm creating this issue to remind myself to fix it.

**Comment by herodev1337** [comment]
Fixed by #10

**Comment by williamjohnstone** [comment]
Resolved by #10
